src/platform/imx8mp-verdin/inc/plat/psci.h

```diff
@@ -0,0 +1,18 @@
+/**
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright (c) Bao Project and Contributors. All rights reserved.
+ */
+
+#ifndef __PLAT_PSCI_H__
+#define __PLAT_PSCI_H__
+
+#define PSCI_POWER_STATE_LVL_0    0x0000000
+#define PSCI_POWER_STATE_LVL_1    0x1000000
+#define PSCI_POWER_STATE_LVL_2    0x2000000
+#define PSCI_STATE_TYPE_STANDBY   0x00000
+#define PSCI_STATE_TYPE_BIT       (1UL << 16)
+#define PSCI_STATE_TYPE_POWERDOWN PSCI_STATE_TYPE_BIT
+
+#define PLAT_PSCI_POWERDOWN_NOT_SUPPORTED
+
+#endif // __PLAT_PSCI_H__
```

Comment:
```suggestion
#endif // PLAT_PSCI_H
```

Reply:
Addressed [here](https://github.com/bao-project/bao-hypervisor/compare/32a5a3d596aaeb38279092090d8c0334935055cc..db2d1e34a81c62020cccff58d7bcffeb93269c58) for all discussions